_includes/sql/v1.1/functions.md

```diff
@@ -150,6 +150,17 @@ Function &rarr; Returns | Description
 <code>translate(input: <a href="string.html">string</a>, find: <a href="string.html">string</a>, replace: <a href="string.html">string</a>) &rarr; <a href="string.html">string</a></code> | <span class="funcdesc">In `input`, replaces the first character from `find` with the first character in `replace`; repeat for each character in `find`. <br/><br/>For example, `translate('doggie', 'dog', '123');` returns `1233ie`.</span>
 <code>upper(val: <a href="string.html">string</a>) &rarr; <a href="string.html">string</a></code> | <span class="funcdesc">Converts all characters in `val`to their to their upper-case equivalents.</span>
 
+### Array Functions
```

Comment:
This doesn't mention `||` as an append operator. All of its features can be used by using one of the functions below, but IMO it's the most pleasant way to work with arrays and we should probably highlight it somewhere. I guess it doesn't go here with the functions though, not sure of the right place for it.